MODULE_REPORT.md

```diff
@@ -0,0 +1,28 @@
+## Module Report
+### Unknown Global
+
+**Global**: `Ember.testing`
+
+**Location**: `addon/services/app-version.js` at line 21
+
+```js
+
+  isNewer: computed("latestVersion", function() {
+    if (Ember.testing === true || addonConfig.isEnabled === false) {
+      return false;
+    }
+```
+
+### Unknown Global
+
+**Global**: `Ember.testing`
+
+**Location**: `addon/services/app-version.js` at line 28
+
+```js
+
+  start() {
+    if (Ember.testing === true || addonConfig.isEnabled !== true) {
+      return;
+    }
+```
```

Comment:
I dont think this file needs to be included. Its telling us that there is an error - `Unknown Global: Ember.testing`

Should remove this file, but if this is still an error need to sort that too.